composer.json

```diff
@@ -21,33 +21,48 @@
     ],
     "require": {
         "php": "^7.2",
-        "sylius/sylius": "1.3.*@dev"
+        "sylius/sylius": "^1.3",
+        "platformsh/symfonyflex-bridge": "^1.3",
+        "sensiolabs/security-checker": " ^4.1"
     },
     "require-dev": {
-        "behat/behat": "^3.2",
-        "behat/mink": "^1.7",
+        "behat/behat": "^3.4",
+        "behat/mink": "^1.7@dev",
         "behat/mink-browserkit-driver": "^1.3",
         "behat/mink-extension": "^2.2",
         "behat/mink-selenium2-driver": "^1.3",
-        "bossa/phpspec2-expect": "^1.0",
-        "coduo/php-matcher": "^2.1",
-        "friends-of-behat/context-service-extension": "^1.0",
-        "friends-of-behat/cross-container-extension": "^1.0",
-        "friends-of-behat/exclude-specifications-extension": "^0.1",
-        "friends-of-behat/performance-extension": "^1.0",
+        "friends-of-behat/context-service-extension": "^1.2",
+        "friends-of-behat/cross-container-extension": "^1.1",
+        "friends-of-behat/page-object-extension": "^0.1",
         "friends-of-behat/service-container-extension": "^1.0",
         "friends-of-behat/suite-settings-extension": "^1.0",
-        "friends-of-behat/symfony-extension": "^1.0",
-        "friends-of-behat/variadic-extension": "^1.0",
-        "lakion/api-test-case": "^1.0",
+        "friends-of-behat/symfony-extension": "^1.2.1",
+        "friends-of-behat/variadic-extension": "^1.1",
+        "lakion/api-test-case": "^3.0",
         "lakion/mink-debug-extension": "^1.2.3",
-        "phpspec/phpspec": "^3.2",
-        "phpunit/phpunit": "^5.6",
-        "se/selenium-server-standalone": "^2.52",
-        "sensio/generator-bundle": "^3.1",
-        "stripe/stripe-php": "^4.1"
+        "phpspec/phpspec": "^4.0",
+        "phpunit/phpunit": "^6.5",
+        "stripe/stripe-php": "^4.1",
+        "sylius-labs/coding-standard": "^2.0",
+        "symfony/browser-kit": "^3.4|^4.1",
+        "symfony/debug-bundle": "^3.4|^4.1",
+        "symfony/dotenv": "^3.4|^4.1",
+        "symfony/flex": "^1.1",
+        "symfony/intl": "^3.4|^4.1",
+        "symfony/web-profiler-bundle": "^3.4|^4.1",
+        "symfony/web-server-bundle": "^3.4|^4.1"
     },
     "prefer-stable": true,
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "App\\Tests\\": "tests/"
```

Comment:
Should it really be `App\\Tests\\` not `Tests\\App\\`?

Reply:
Since I've followed the pattern from Sylius-Standard, I guess it's a question for @pamil

Reply:
cc @pamil

Reply:
cc https://github.com/symfony/demo/blob/master/composer.json#L61

Reply:
👍